README.md

```diff
@@ -2,16 +2,29 @@
 
 ImageLab is a framework that allows students to develop
 image modification processors (filters) and to experience
-the results visually and aurally.
+the results visually and aurally. This project can be ran as a BlueJ Project, directly from the command line, or built with a gradle wrapper. 
 
-## Contents
+Project Website & User Instructions: https://metrocs.github.io/imagelab/
+_____
+
+## Table of Contents
++ [Repository Contents](#repository-contents)
++ [Build Instructions](#build-instructions)
++ [License](#license)
+
+## Repository Contents 
 - The __imagelab__ and __sound__ packages may be provided in an `imagelab.jar` file.
 - The `imagelab.jar` file also includes the class __Run__ with a `public static void main` method in the unnamed package, used to facilitate running the ImageLab application.
-- The repository contents may also be used directly as a [BlueJ](http://bluej.org) project.
 - The __filter__ package is generally provided as a folder with source (`.java`) and compiled (`.class`) files, with the source versions serving as examples for students to create their own filters.
 - A sample set of images are provided in an __"images"__ directory.
 
-## Creating an executable jar file
+## Build Instructions 
+
+### Build System Requirements
+- Install Gradle
+  * Gradle is the tool used in this repository for automation of the build. Please visit https://gradle.org/ for specific installation instructions.
```

Comment:
Seeing this here, how about no substructure under the ### heading? Something like the following two sentences without any bullet structure:

The [Gradle](https://gradle.org) tool is used for build automation. Please see https://gradle.org for installation and operation instructions.